README.md

```diff
@@ -4,7 +4,7 @@
 
 An extension to [SnapshotTesting](https://github.com/pointfreeco/swift-snapshot-testing) which allows you to create images combining the output of multiple snapshot strategies, assuming they all output to `UIImage`.
 
-In essence, this allows you to have a single image which represents a single view in your application, shown in multiple different configurations. This might be useful, for example, where you want to visualise the same UIViewController on multiple devices or in light and dark mode.
+In essence, this allows you to have a single image which represents a single view in your application, shown in multiple different configurations. This might be useful, for example, where you want to visualize the same UIViewController on multiple devices or in light and dark mode.
```

Comment:
Ahh, this (and many other examples here) isn't actually a typo but just American English vs British English. I'm British so default to my locale 🙂

Reply:
Apology my friend. I was following [this popular convention](https://github.com/kodecocodes/swift-style-guide?tab=readme-ov-file#language). You can definitely have your own native style 🙌🏻